recipes/gimmemotifs/build.sh

```diff
@@ -1,2 +1,3 @@
 #!/bin/bash
+sed -i 's/"gcc"/os.environ["GCC"]/' compile_externals.py
```

Comment:
In this case it is probably easier to create a `.patch` file for those changes. Also, please make sure the `CFLAGS` and `LDFLAGS` are considered, too.

Reply:
yeah, it's a very simple compilation so this is enough..

Reply:
Even if it's just a simple one, I'd still recommend using those flags for compatibility's sake.

If you want to inspect which additional parameters are set, you can inspect the build log:
```
11:18:07 BIOCONDA INFO (OUT) INFO: activate-gcc_linux-64.sh made the following environmental changes:
11:18:07 BIOCONDA INFO (OUT) +CC=$BUILD_PREFIX/bin/x86_64-conda_cos6-linux-gnu-cc
11:18:07 BIOCONDA INFO (OUT) +CFLAGS=-march=nocona -mtune=haswell -ftree-vectorize -fPIC -fstack-protector-strong -fno-plt -O2 -ffunction-sections -pipe -I$PREFIX/include -fdebug-prefix-map=$SRC_DIR=/usr/local/src/conda/gimmemotifs-0.13.1 -fdebug-prefix-map=$PREFIX=/usr/local/src/conda-prefix
11:18:07 BIOCONDA INFO (OUT) +CMAKE_PREFIX_PATH=:$PREFIX:$BUILD_PREFIX/x86_64-conda_cos6-linux-gnu/sysroot/usr
11:18:07 BIOCONDA INFO (OUT) +_CONDA_PYTHON_SYSCONFIGDATA_NAME=_sysconfigdata_x86_64_conda_cos6_linux_gnu
11:18:07 BIOCONDA INFO (OUT) +CPPFLAGS=-DNDEBUG -D_FORTIFY_SOURCE=2 -O2 -I$PREFIX
11:18:07 BIOCONDA INFO (OUT) +CPP=$BUILD_PREFIX/bin/x86_64-conda_cos6-linux-gnu-cpp
11:18:07 BIOCONDA INFO (OUT) +DEBUG_CFLAGS=-march=nocona -mtune=haswell -ftree-vectorize -fPIC -fstack-protector-all -fno-plt -Og -g -Wall -Wextra -fvar-tracking-assignments -ffunction-sections -pipe -I$PREFIX/include -fdebug-prefix-map=$SRC_DIR=/usr/local/src/conda/gimmemotifs-0.13.1 -fdebug-prefix-map=$PREFIX=/usr/local/src/conda-prefix
11:18:07 BIOCONDA INFO (OUT) +DEBUG_CPPFLAGS=-D_DEBUG -D_FORTIFY_SOURCE=2 -Og -I$PREFIX
11:18:07 BIOCONDA INFO (OUT) +GCC_AR=$BUILD_PREFIX/bin/x86_64-conda_cos6-linux-gnu-gcc-ar
11:18:07 BIOCONDA INFO (OUT) +GCC_NM=$BUILD_PREFIX/bin/x86_64-conda_cos6-linux-gnu-gcc-nm
11:18:07 BIOCONDA INFO (OUT) +GCC=$BUILD_PREFIX/bin/x86_64-conda_cos6-linux-gnu-gcc
11:18:07 BIOCONDA INFO (OUT) +GCC_RANLIB=$BUILD_PREFIX/bin/x86_64-conda_cos6-linux-gnu-gcc-ranlib
11:18:07 BIOCONDA INFO (OUT) +LDFLAGS=-Wl,-O2 -Wl,--sort-common -Wl,--as-needed -Wl,-z,relro -Wl,-z,now -Wl,--disable-new-dtags -Wl,--gc-sections -Wl,-rpath,$PREFIX/lib -Wl,-rpath-link,$PREFIX/lib -L$PREFIX/lib
```

```
11:18:08 BIOCONDA INFO (OUT) running build_ext
11:18:08 BIOCONDA INFO (OUT) building 'gimmemotifs.c_metrics' extension
11:18:08 BIOCONDA INFO (OUT) creating build/temp.linux-x86_64-3.6
11:18:08 BIOCONDA INFO (OUT) creating build/temp.linux-x86_64-3.6/gimmemotifs
11:18:08 BIOCONDA INFO (OUT) $BUILD_PREFIX/bin/x86_64-conda_cos6-linux-gnu-cc -DNDEBUG -fwrapv -O2 -Wall -Wstrict-prototypes -march=nocona -mtune=haswell -ftree-vectorize -fPIC -fstack-protector-strong -fno-plt -O2 -ffunction-sections -pipe -I$PREFIX/include -fdebug-prefix-map=$SRC_DIR=/usr/local/src/conda/gimmemotifs-0.13.1 -fdebug-prefix-map=$PREFIX=/usr/local/src/conda-prefix -DNDEBUG -D_FORTIFY_SOURCE=2 -O2 -I$PREFIX -fPIC -I$PREFIX/include/python3.6m -c gimmemotifs/c_metrics.c -o build/temp.linux-x86_64-3.6/gimmemotifs/c_metrics.o
11:18:08 BIOCONDA INFO (OUT) gimmemotifs/c_metrics.c:285:5: warning: function declaration isn't a prototype [-Wstrict-prototypes]
11:18:08 BIOCONDA INFO (OUT)  int get_truncate_len(len1, len2, pos) {
11:18:08 BIOCONDA INFO (OUT)      ^~~~~~~~~~~~~~~~
11:18:08 BIOCONDA INFO (OUT) gimmemotifs/c_metrics.c: In function 'get_truncate_len':
11:18:08 BIOCONDA INFO (OUT) gimmemotifs/c_metrics.c:285:5: warning: type of 'len1' defaults to 'int' [-Wimplicit-int]
11:18:08 BIOCONDA INFO (OUT) gimmemotifs/c_metrics.c:285:5: warning: type of 'len2' defaults to 'int' [-Wimplicit-int]
11:18:08 BIOCONDA INFO (OUT) gimmemotifs/c_metrics.c:285:5: warning: type of 'pos' defaults to 'int' [-Wimplicit-int]
11:18:08 BIOCONDA INFO (OUT) gimmemotifs/c_metrics.c: At top level:
11:18:08 BIOCONDA INFO (OUT) gimmemotifs/c_metrics.c:864:15: warning: initialization makes integer from pointer without a cast [-Wint-conversion]
11:18:08 BIOCONDA INFO (OUT)   {NULL, NULL, NULL, 0, NULL}
11:18:08 BIOCONDA INFO (OUT)                ^~~~
11:18:08 BIOCONDA INFO (OUT) gimmemotifs/c_metrics.c:864:15: note: (near initialization for 'CoreMethods[4].ml_flags')
11:18:08 BIOCONDA INFO (OUT) gimmemotifs/c_metrics.c:864:24: warning: excess elements in struct initializer
11:18:08 BIOCONDA INFO (OUT)   {NULL, NULL, NULL, 0, NULL}
11:18:08 BIOCONDA INFO (OUT)                         ^~~~
11:18:08 BIOCONDA INFO (OUT) gimmemotifs/c_metrics.c:864:24: note: (near initialization for 'CoreMethods[4]')
11:18:08 BIOCONDA INFO (OUT) x86_64-conda_cos6-linux-gnu-gcc -pthread -shared -Wl,-O2 -Wl,--sort-common -Wl,--as-needed -Wl,-z,relro -Wl,-z,now -Wl,-rpath,$PREFIX/lib -L$PREFIX/lib -Wl,-O2 -Wl,--sort-common -Wl,--as-needed -Wl,-z,relro -Wl,-z,now -Wl,-rpath,$PREFIX/lib -L$PREFIX/lib -Wl,-O2 -Wl,--sort-common -Wl,--as-needed -Wl,-z,relro -Wl,-z,now -Wl,--disable-new-dtags -Wl,--gc-sections -Wl,-rpath,$PREFIX/lib -Wl,-rpath-link,$PREFIX/lib -L$PREFIX/lib -march=nocona -mtune=haswell -ftree-vectorize -fPIC -fstack-protector-strong -fno-plt -O2 -ffunction-sections -pipe -I$PREFIX/include -fdebug-prefix-map=$SRC_DIR=/usr/local/src/conda/gimmemotifs-0.13.1 -fdebug-prefix-map=$PREFIX=/usr/local/src/conda-prefix -DNDEBUG -D_FORTIFY_SOURCE=2 -O2 -I$PREFIX build/temp.linux-x86_64-3.6/gimmemotifs/c_metrics.o -o build/lib.linux-x86_64-3.6/gimmemotifs/c_metrics.cpython-36m-x86_64-linux-gnu.so
```